specification/ml/_types/TrainedModel.ts

```diff
@@ -327,13 +373,16 @@ export class TrainedModelAssignmentTaskParameters {
    * @availability stack since=8.4.0
    * @availability serverless
    */
-  cache_size: ByteSize
+  cache_size?: ByteSize
```

Comment:
I can't comment on `model_bytes` above, but it's a long, not an integer.

Reply:
probably a ByteSize then